api/otp.go

```diff
@@ -135,3 +141,84 @@ func (a *API) shouldCreateUser(r *http.Request, params *OtpParams) bool {
 	}
 	return true
 }
+
+// GetPhoneOtp save and return the OTP code
+func (a *API) GetPhoneOtp(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Can we find a way to reuse the `Otp` method above? That way, we won't have to maintain 2 separate functions with almost identical functionality.